### PR TITLE
add forward transfer view, add forward transfer api

### PR DESCRIPTION
**Pull request by akrckn**
- [x] add forward transfer rpc api method
- [x] add forward transfer api method
- [x] move forward transfer api to `block` route
  - [x] remove 0x prefix in to address
  - [x] implement decimal representation of value
- [x] simplify duplicate forward transfer filtering

TODO:

- [x] add more tests
- [x] Seems like we are not getting the correct address in forward transfer object or it is falsely formatted

### CODE REVIEW

File: sdk/src/main/java/com/horizen/account/api/rpc/types/ForwardTransfersView.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                value = String.valueOf(weiValue);
+            } else {
+                to = Numeric.toHexString(address.address());
+                value = Numeric.toHexStringWithPrefix(weiValue);
+            }
+            forwardTransfers.put(i, new ForwardTransferData(to, value));
+        }
+    }
+
+    public TreeMap<Integer, ForwardTransferData> getForwardTransfers() {
```
Comment (i-Alex) [inline sdk/src/main/java/com/horizen/account/api/rpc/types/ForwardTransfersView.java:39]: why do we need to use Map here?
We pass `List<ForwardTransfer> transactions` -> so the result can be List<ForwardTransferData> to be sure that the result has a proper order of FTs.

**Review by i-Alex — APPROVED** [review]

**Review by alsala — APPROVED** [review]

File: sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala [hunk 2]
```diff
@@ -384,4 +385,15 @@ class EthService(val scNodeViewHolderRef: ActorRef, val nvtimeout: FiniteDuratio
       }
     }
   }
+
+  @RpcMethod("eth_getForwardTransfers")
```
Comment (JcBernack) [inline sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala:389]: Actually, I think we should call this `zen_getForwardTransfers`.
Moreover, the idea was to have all the `eth_` methods in `EthService` not *all* RPC methods - but that is refactoring for later.

File: sdk/src/main/scala/com/horizen/account/api/http/AccountTransactionApiRoute.scala [hunk 3]
```diff
@@ -4,7 +4,7 @@ import org.web3j.crypto._
 import akka.actor.{ActorRef, ActorRefFactory}
 import akka.http.scaladsl.server.Route
 import akka.pattern.ask
-import com.fasterxml.jackson.annotation.JsonView
+import com.fasterxml.jackson.annotation.{JsonView}
```
Comment (JcBernack) [inline sdk/src/main/scala/com/horizen/account/api/http/AccountTransactionApiRoute.scala:7]: press `optimize imports` :wink:

**Review by JcBernack — APPROVED** [review]

File: sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala (hunk 2 above)
Comment (i-Alex) [reply sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala:389]: so far, let's keep it as is.
Agree to keep the new methods with another prefix.

**Pull request merged**